test/data/testSP.py

```diff
@@ -447,6 +446,12 @@ class TurbomoleSPTest(GenericSPTest):
     """Customized restricted single point unittest"""
 
     num_scf_criteria = 2
+    
+    def testmetadata_basis_set(self):
+        """Does metadata have expected keys and values?"""
+        # One of our test cases used sto-3g hondo
+        valid_basis = self.data.metadata["basis_set"].lower() == "sto-3g" or self.data.metadata["basis_set"].lower() == "sto-3g hondo"

```

Comment:
```suggestion
        valid_basis = self.data.metadata["basis_set"].lower() in ("sto-3g", "sto-3g hondo")
```
though in the future, we should map `sto-3g hondo` to `sto-3g`, since AFAIK there is only one version of STO-3G.